net-misc/bird/bird-2.0.7.ebuild

```diff
@@ -1,28 +1,36 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
+inherit eutils
+
 DESCRIPTION="A routing daemon implementing OSPF, RIPv2 & BGP for IPv4 & IPv6"
-HOMEPAGE="http://bird.network.cz"
+HOMEPAGE="https://bird.network.cz"
 SRC_URI="ftp://bird.network.cz/pub/${PN}/${P}.tar.gz"
 LICENSE="GPL-2"
 
 SLOT="0"
 KEYWORDS="~amd64 ~arm64 ~x86 ~x64-macos"
-IUSE="+client debug"
+IUSE="+client debug libssh"
 
 RDEPEND="client? ( sys-libs/ncurses )
-	client? ( sys-libs/readline )"
+	client? ( sys-libs/readline )
```

Comment:
Please add := to all deps we link against that also have a subslot

Reply:
I didn’t spotted that, it’s fixed now, thanks :)